services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -72,20 +78,25 @@ const defaultProps = {
 };
 
 export class NoticeOfWorkApplication extends Component {
-  state = {
-    currentStep: 0,
-    isLoaded: false,
-    isImported: false,
-    isNoWLoaded: false,
-    associatedMineGuid: "",
-    isViewMode: true,
-    showOriginalValues: false,
-    fixedTop: false,
-    menuVisible: false,
-    isDecision: false,
-    buttonValue: "REV",
-    buttonLabel: "Technical Review",
-  };
+  static noticeOfWorkPageFromRoute = {};
+
+  constructor(props) {
+    super(props);
```

Comment:
This change isn't necessary unless you are using the constructor for something else, if not, initializing as 'state = {}` is consistent throughout the app.